src/iiiftilesource.js

```diff
@@ -204,13 +207,15 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
         if ( !$.isPlainObject(data) ) {
             var options = configureFromXml10( data );
             options['@context'] = "http://iiif.io/api/image/1.0/context.json";
-            options['@id'] = url.replace('/info.xml', '');
+            options._id = url.replace('/info.xml', '');
```

Comment:
The `configure` function, while it looks like it's just another member function, is actually called like a static function. It's called before the TileSource object is created, and it sets up the options that go into the TileSource constructor. Because of that, we don't want `_id` here, since the IIIFTileSource constructor doesn't accept _id (nor should it, since it's internal). I figure we might as well stick with `@id` as before, but it's fine to switch over to `id` if you prefer. 

Same comment for the rest of this function, of course.